### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by Buyaki01**
- Implement the design from the sneak peek wireframes. Styling is nice to have but focus on making sure everything functions first
- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md)
- Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data)

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+            <h4> <%= comment.author.name %>: <%= comment.text %> </h4>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
+  <% end %>
+
+  <div class="d-flex justify-content-center mt-3 p-2">
+    <button> See all posts </button>
+  </div>
```
Comment (Emmyn5600) [inline app/views/posts/index.html.erb]: - [ ] **See all Post** links open user's post index as displayed on this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png), kindly implement it.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+      <div class="row">
+        <div class="col-3 border"> 
+          <%= image_tag(user.photo) %> 
+        </div>
+        <div class="col-8 offset-1 border">
+          <h2> <%= user.name %> </h2>
+          <h6 class="d-flex justify-content-end"> Number of posts: <%= user.posts_counter || 0 %>  </h6>
+        </div>
+      </div>
+    <% end %>
```
Comment (Emmyn5600) [inline app/views/users/index.html.erb:13]: - [ ] Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+    <p> <%= @user.bio %></p>
+  </div>
+  
+  <% @user.fetch_three_recent_posts.each do |post| %>
+    <div class="border mt-3 p-2">
+      <h3> <%=post.title%> </h3>
+      <p> <%=post.text%> </p>
+      <h6 class="d-flex justify-content-end"> Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter%> </h6>
+    </div>
+  <% end %>
```
Comment (Emmyn5600) [inline app/views/users/show.html.erb]: - [ ] When you click on the post you have to see all the comments that have been commented on each post. Kindly follow the [design](
https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) to implement it.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Buyaki01

Good job so far! I know you have worked hard on making this milestone however, There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good PR description :heavy_check_mark:


## Required Changes :recycle:

- [ ]  Check the comment under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @Buyaki01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**